src/js/file-manager.js

```diff
@@ -163,4 +163,50 @@ fileManager.isTooLarge = () => { return false; };
  */
 fileManager.getMaxSizeReadable = () => { return `${5}MB`; };
 
+/**
+ * @function resizeImage
+ *
+ * @param {File} file - image file to be resized
+ * @param {number} maxWidth - maximum width of resized image
+ * @param {number} maxHeight - maximum height of resized image
+ *
+ * @return {Promise<Blob>} promise of resized image blob
+ */
+fileManager.resizeImage = ( file, maxWidth, maxHeight ) => {
+    return new Promise( ( resolve, reject ) => {
+        let image = new Image();
+        image.src = URL.createObjectURL( file );
+        image.onload = () => {
+            let width = image.width;
+            let height = image.height;
+
+            if ( width <= maxWidth && height <= maxHeight ) {
+                resolve( file );
+            }
+
+            let newWidth;
+            let newHeight;
+
+            if ( width > height ) {
```

Comment:
If I understand correctly, this means that the two parameters of this function cannot actually be used. That is fine, as we don't need 2. So I think it would simplify a bit by changing this function to only have 1 `maxPixel` parameter.

Reply:
Done